apps/web/__e2e__/editor.test.ts

```diff
@@ -84,6 +84,87 @@ test("creating a new note should clear the editor contents & title", async ({
   expect(await notes.editor.getContent("text")).toBe("");
 });
 
+test("when autosave is disabled, pressing ctrl+s should save the note", async ({
+  page
+}) => {
+  const app = new AppModel(page);
+  await app.goto();
+  const notes = await app.goToNotes();
+  const content = "a ".repeat(100);
+  await notes.createNote({
+    title: NOTE.title,
+    content
+  });
+
+  await page.locator(getTestId("editor-save-state-notsaved")).waitFor();
```

Comment:
Let's make getting the save state of the editor a method in `editor-model.ts` since it'll be reused.